cassandra/policies.py

```diff
@@ -749,6 +749,18 @@ class WriteType(object):
     A lighweight-transaction write, such as "DELETE ... IF EXISTS".
     """
 
+    VIEW = 6
+    """
+    This WriteType is only seen in results for requests that were unable to
+    complete MV operations.
+    """
+
+    CDC = 7
+    """
+    This WriteType is only seen in results for requests that were unable to
+    complete CDC operations.
+    """
+
```

Comment:
There types should probably be added in WriteType.name_to_value:  https://github.com/datastax/python-driver/blob/24961b6bb3d978ee4d328374c50db75115280258/cassandra/policies.py#L765

Otherwise, the WriteFailureMessage class wont be able to use them at all: https://github.com/datastax/python-driver/blob/24961b6bb3d978ee4d328374c50db75115280258/cassandra/protocol.py#L319